Follow-up: Add pod integration auto-enablement info to documentation (#7167)
kubernetes-sigs/kueue: site/content/en/docs/tasks/run/deployment.md

1. Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version).

2. Ensure that you have the `deployment` integration enabled, for example:
   ```yaml
   apiVersion: config.kueue.x-k8s.io/v1beta1
   kind: Configuration
   integrations:
     frameworks:
      - "deployment"
   ```
   {{% alert title="Pod integration requirements" color="primary" %}}

>    Since Kueue v0.15, `"pod"` integration is automatically enabled for `"deployment"` objects. To manage plain `"pod"`s directly, you still need to explicitly enable the `"pod"` integration.

Reviewer: ```suggestion
   Since Kueue v0.15, you don't need to explicitly enable "pod" integration to use the "deployment" integration. 
```
Maybe that is shorter and focuses more on he user

Author: Done.
